docs/configuration/configuration.md

```diff
@@ -1294,6 +1294,9 @@ url: <string>
 write_relabel_configs:
   [ - <relabel_config> ... ]
 
+# Name of the remote write config, useful for users who have multiple remote writes configured.
```

Comment:
This is a bit vague, what does it actually do?

Reply:
Just that, a name for the remote write queue. If you don't set one it defaults to a hash of that remote write config. It's useful for users who have multiple remote writes, especially if they have more than one remote write to the same endpoint but with different relabel configs for example. I can include that last bit in the comment here?

Reply:
That still doesn't explain what it actually does, only the general intention behind it. You need to mention where this value will actually appear.

Reply:
It's strictly for identification (by the user, not internally within remote write) of a given queue. So the name would show up in a field during structured logging, a label in remote write metrics, etc.

Reply:
If it has a semantic effect (and metric labels count) that needs to be mentioned here. Otherwise it's unclear what this actually does, it could just be a form of comment that never leaves the config file.

Reply:
updated the comment

Reply:
This is still not quite right for a reference doc. I'd do something like:

Name of the remote write config, which if specified must be unique. This will be used in metrics etc. in place of a generated value to help users distinguish between remote write configs.


This is also not the syntax we use to indicate an optional field.